Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs

```diff
@@ -72,6 +72,20 @@ public override async Task<IActionResult> Edit([FromForm] ProblemAdministrationM
         return response;
     }
 
+    [HttpPost("{problemId:int}")]
+    [ProtectedEntityAction]
+    public async Task<IActionResult> ValidateRetest(int problemId)
+    {
+        if (!await this.problemsDataService.ExistsById(problemId))
+        {
+            return this.UnprocessableEntity();
+        }
+
+        var validationModel = await this.problemsBusinessService.ValidateRetest(problemId);
```

Comment:
This validation should also (or solely) be happening in the Retest method itself, in case the method is called directly.

Reply:
Added to Retest endpoint